src/book/entities/Transaction.ts

```diff
@@ -115,6 +115,9 @@ export async function updateCache(
     isDelete?: boolean,
   },
 ) {
+  queryClient.invalidateQueries({
+    queryKey: [...Transaction.CACHE_KEY, entity.guid],
+  });
```

Comment:
unrelated but bug: Due to us caching each transaction with useQuery, we want to invalidate the transaction so when we load it in the form, it shows as expected (previously it was showing old values)